runtime/src/iree/hal/cts/driver_test.h

```diff
@@ -16,19 +16,44 @@
 #include "iree/testing/gtest.h"
 #include "iree/testing/status_matchers.h"
 
-namespace iree {
-namespace hal {
-namespace cts {
+namespace iree::hal::cts {
 
-class driver_test : public CTSTestBase<> {
+// NOTE: does not use CTSTestBase as we don't want the automatically created
+// driver/device it provides.
+class DriverTest : public ::testing::Test {
```

Comment:
Good idea - lets the other tests reuse devices while this test works as it did before.